plugins/woocommerce-blocks/assets/js/blocks/product-filters/utils/colors/index.ts

```diff
@@ -10,15 +10,15 @@ import { paramCase as kebabCase } from 'change-case';
  * @param {string} value Value of the color.
  * @return {string} CSS variable value.
  */
-function getCSSVar(
+export function getCSSVar(
```

Comment:
This needs more specific naming, as the variable is only for color, while we have more styles, for example, spacing.

Reply:
Great suggestion! I renamed and started reusing this util function in other places: https://github.com/woocommerce/woocommerce/pull/52678/commits/1b2a1ba1a6d9baebcc6a9f86c87ce2b0fdc7f66e